src/utils.c

```diff
@@ -152,3 +156,63 @@ int must_snprintf(char *str, size_t size, const char *format, ...)
 
 	return n;
 }
+
+int sc_nonfatal_mkpath(const char *const path, mode_t mode)
```

Comment:
Yeah, I plan to unify all the names n the C code and this feels like one of the first ones that don't call `di.e.()`